setup.py

```diff
@@ -339,13 +345,13 @@ def _get_package_path(package_name):
 # Parameters parsed from environment
 ################################################################################
 
-VERBOSE_SCRIPT = True
+VERBOSE_SCRIPT = str2bool(os.getenv("VERBOSE", "1"))
```

Comment:
You sure about this one? It's a double whammy: VERBOSE is pretty generic envvar name, and also you have defaulted it to 1 which is unusual.

Reply:
> VERBOSE is pretty generic envvar name

CMake respects `VERBOSE` during build. I think it's okay.

> you have defaulted it to 1 which is unusual

The previous default value is `True`.

------

This one can be disabled by `python setup.py --quiet`, even with `VERBOSE=1`.